src/aol-hb.js

```diff
@@ -27,4 +32,27 @@ globalContext.init = (bidRequestConfig, placementsConfigs) => {
   };
 };
 
+globalContext.que.push = (cmd) => {
+  if (typeof cmd === 'function') {
+    try {
+      cmd.call();
+    } catch (e) {
+      console.warn('Error processing command :' + e.message);
```

Comment:
It would be a good idea to dump the whole error object to the console, the same as you did below, i.e. `console.warn('Error processing command', e);`.